packages/frontier-plugins/plugin-qa/bin/run

```diff
@@ -1,4 +1,11 @@
 #!/usr/bin/env node
+const tsConfig = require('../tsconfig.base.json');
```

Comment:
Whats happening here?

Reply:
I was to remove this. That was from the type aliasing fixes I was trying.